modules/document_repository/php/edit.class.inc

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * This file contains the NDB_Form_media class
+ *
+ * PHP Version 5
+ *
+ * @category Loris
+ * @package  Media
+ * @author   Alex Ilea <ailea.mcin@gmail.com>
```

Comment:
Maybe maintainer instead?

Reply:
You can include multiple @author lines.

Reply:
Well, it's an entirely new file created by @kongtiaowang, with @alisterdev's work used as a template. If there's someone to contact about the code though, I think it would be @kongtiaowang. Unless it's best to just write `Loris Team`.